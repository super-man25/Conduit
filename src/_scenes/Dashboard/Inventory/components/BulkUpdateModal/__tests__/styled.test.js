import React from 'react';
import renderer from 'react-test-renderer';
import {
  Field,
  FieldErrorText,
  Label,
  ModalBody,
  ModalHeader,
  NumberInput,
  Title,
} from '../styled';

describe('BulkUpdateModal styled components', () => {
  it('<Field /> renders correctly', () => {
    const tree = renderer.create(<Field />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('<FieldErrorText /> renders correctly', () => {
    const tree = renderer
      .create(<FieldErrorText>Text</FieldErrorText>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('<Label /> renders correctly', () => {
    const tree = renderer.create(<Label>Label</Label>).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('<ModalBody /> renders correctly', () => {
    const tree = renderer.create(<ModalBody>Modal Body</ModalBody>).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('<ModalHeader /> renders correctly', () => {
    const tree = renderer
      .create(<ModalHeader>Modal Header</ModalHeader>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('<NumberInput /> renders correctly', () => {
    const tree = renderer
      .create(<NumberInput type="number" placeholder="$ Manual Price" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('<Title /> renders correctly', () => {
    const tree = renderer.create(<Title>Title</Title>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
