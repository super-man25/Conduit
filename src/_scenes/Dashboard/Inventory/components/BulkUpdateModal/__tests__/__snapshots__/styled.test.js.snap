// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BulkUpdateModal styled components <Field /> renders correctly 1`] = `
.c0 {
  position: relative;
}

<div
  className="sc-fzpkqZ sc-fznAgC c0"
/>
`;

exports[`BulkUpdateModal styled components <FieldErrorText /> renders correctly 1`] = `
.c0 {
  color: #070707;
  margin: 0;
  position: absolute;
  top: 100%;
}

<p
  className="sc-fznLPX sc-fzoaKM c0"
>
  Text
</p>
`;

exports[`BulkUpdateModal styled components <Label /> renders correctly 1`] = `
.c0 {
  display: block;
  margin-bottom: 0.5rem;
  color: #070707;
}

<label
  className="c0"
>
  Label
</label>
`;

exports[`BulkUpdateModal styled components <ModalBody /> renders correctly 1`] = `
.c0 {
  padding: 1.5rem;
  background-color: #FFFFFF;
  border-bottom-left-radius: 3px;
  border-bottom-right-radius: 3px;
}

<div
  className="c0"
>
  Modal Body
</div>
`;

exports[`BulkUpdateModal styled components <ModalHeader /> renders correctly 1`] = `
.c0 {
  padding: 1.5rem;
  background-color: #F8F8F8;
  border-bottom: 1px solid #0A169A;
  border-top-left-radius: 3px;
  border-top-right-radius: 3px;
}

<div
  className="c0"
>
  Modal Header
</div>
`;

exports[`BulkUpdateModal styled components <NumberInput /> renders correctly 1`] = `
.c0 {
  background-color: #FFFFFF;
  border: 1px solid #999999;
  border-radius: 3px;
  height: 40px;
  margin: 0;
  padding: 0 0.5rem;
  width: 100%;
}

<input
  className="c0"
  placeholder="$ Manual Price"
  type="number"
/>
`;

exports[`BulkUpdateModal styled components <Title /> renders correctly 1`] = `
.c0 {
  color: #070707;
  font-size: 18px;
  font-weight: normal;
  margin: 0;
  margin-bottom: 0.25rem;
  color: #070707;
}

<h4
  className="sc-fzoant sc-fzomME c0"
  color="#070707"
>
  Title
</h4>
`;
